WebContent/js/master.js

```diff
@@ -319,6 +319,26 @@ function HtmlEncode(s) {
 	return s;
 }
 
+function addValidators() {

```

Comment:
Instead of a function here called `addValidators` that we then need to call from every page, can't we just add this to the `ready` function that gets called automatically?

https://github.com/StarExec/StarExec/blob/669e638fa1608fa0c59c3fb2a369e1301e8a3488/WebContent/js/master.js#L11-L12

Reply:
Let's address this later. No reason for this to block this PR.